src/chrome/content/rules/stinpriza.org.xml

```diff
@@ -0,0 +1,18 @@
+<ruleset name="*.stinpriza.org">
```

Comment:
You can just name it `stinpriza.org`, otherwise it will look a bit odd.

Please add a comment on the top for non-working domains:
```
Mismatching cert:
ecoefficiency.stinpriza.org

Self signed:
intefix.stinpriza.org
eepf.intefix.stinpriza.org

Reply:
you're right, sorry for the fuzz. 
i will come up with a complete ruleset, after doing a bit of cleanup on unused subdomains.